### DISCUSSION TITLE
Tablet area settings adjustment

### DISCUSSION THREAD

**Discussion by andrey6498** [ppy/osu#17851] [Ideas] [answered]
I think that base increment of 5 for for arrow in area settings is too high there, and ctrl/shift etc modifiers don't work as well, so it's rather problematic to configure while using tablet, since cursor fluctates, while doing this, and arrows for modifying value are useless for this cause because of high base increment.

I'd like to suggest that base arrow increment should be 1 while with ctrl and shift modifiers it should increment by 5 and 10 respectively (or other values you see fit)

**Comment by frenzibyte** [comment, marked as answer]
Should improve with https://github.com/ppy/osu/discussions/16932
